content/hardware/04.pro/boards/portenta-x8/tutorials/01.user-manual/content.md

```diff
@@ -1,1440 +1,1362 @@
+
+Open your browser and navigate to [www.arduino.cc/start](www.arduino.cc/start). The following page will be displayed.
+
+![First Setup](assets/registration-homepage.png "First Setup")
+
+Click on **Linux boards** and sign in to your Arduino Cloud account. If you do not have an account, create a new one from the same webpage.
+
+![Sign into Arduino Cloud](assets/registration-signin.png "Sign into Arduino Cloud")
+
+When successfully logged in, you will be asked to download the **Arduino Create Agent** if you have not done so yet. Click on **DOWNLOAD**.
+
+![Create Agent Installation](assets/agent-installation.png "Create Agent Installation")
+
+The agent will be installed on your computer. This activity might take a few minutes. Once installed, your Portenta X8 will be automatically detected by your PC after a few seconds.
+
+![Portenta X8 successfully detected](assets/board-detection.png "Portenta X8 successfully detected")
+
+Now click on **START CONFIGURATION**. The tool will install all the required add-ons to make your Portenta X8 work efficiently with your PC via serial communication.
+
+You can now proceed to the setup of the board connectivity by clicking **OK, GOT IT**. 
+
+![Arduino Linux Wizard Connectivity Configuration](assets/ootb-wifi-config.png "Arduino Linux Wizard Connectivity Configuration")
+
+***If you face any issue with this flow or prefer to interact directly with your Portenta X8 through the command line, please refer to [this section](#working-with-linux) to learn how to connect with the board leveraging ADB service.***
+
+Once the setup is complete, the browser page will automatically switch to the Arduino Linux Wizard page.
+
+The Arduino Linux Wizard page is accessible at [**http://localhost:8080**](http://localhost:8080). You can use this link to open the Arduino Linux Wizard dashboard anytime.
+
+![Arduino Linux Wizard Home Screen](assets/OOTB_homepage_init.png "Arduino Linux Wizard Home Screen")
+
+The **SYSTEM INFO** button is located at the bottom left corner of the Arduino Linux Wizard page.
+
+![Arduino Linux Wizard System Info](assets/OOTB_system_info.png "Arduino Linux Wizard System Info")
+
+This section displays details about the Portenta X8, such as hardware and software specifications, network status, and active containers.
+
+If you disconnect the Portenta X8 and need to reopen the Arduino Linux Wizard page, use the command line and run the TCP port forwarding command:
+
+```bash
+adb forward tcp:8080 tcp:80
+```
+
+Then, access the page with [**http://localhost:8080**](http://localhost:8080) to open the Arduino Linux Wizard dashboard.
+
+#### Wi-Fi® Configuration
+
+Click **Wi-Fi® Connection** to start configuring your network connectivity. Otherwise, you can connect your Portenta X8 to the Internet through an Ethernet cable using a USB-C® hub with an RJ45 port or a Portenta Carrier. In this tutorial, Wi-Fi® connectivity will be used.
```

Comment:
Let's mention that you can connect the board also via cellular connectivity with a Portenta Carrier and 4G Module.

Reply:
Updated!